README.md

```diff
@@ -10,6 +10,28 @@ The content for the site lives in the `content/` folder. Fork this repo and send
 - Updated course information
 - Anything you can think of
 
+Compiling site locally
+----------------------
+
+Install [Lektor](https://www.getlektor.com/)
+
+```bash
+# Clone the repo
+git clone https://github.com/jackharrhy/Website.git`
```

Comment:
I'd suggest you change this so it points to the computer science society's copy of the repo, and also remove the trailing ` unless it does something that I'm not aware of.

Reply:
Developed this branch on a fork, and completely forgot it was a fork... 🤦‍♂️  
5e1f68e